sda-commons-server-auth/README.md

```diff
@@ -147,6 +147,14 @@ In this case, the `AUTH_KEYS` variable should contain a JSON array of
 ]
 ```
 
+### Periodical reloading of public keys
+
+The public keys are cached locally. For keys with an unknown key-id (kid), 
+the cache is refreshed, and the key is searched again afterwards.<br/>
+In addition, the key-cache is refreshed periodically (currently each 5 min) to remove keys
+which are revoked by their issuers. <br/>
+If the refresh fails (e.g. key-source unavailable, network-problems etc.), the old keys in the cache stay valid.
+
```

Comment:
NIT: You don't need `<br/>` in Markdown, two spaces at the end of the line are enough to produce a line break. But in this case, a single paragraph without line breaks should be fine as well.

```suggestion
The public keys are cached locally. For keys with an unknown key-id (kid), 
the cache is refreshed, and the key is searched again afterwards.
In addition, the key-cache is refreshed periodically (currently each 5 min) to remove keys
which are revoked by their issuers.
If the refresh fails (e.g. key-source unavailable, network-problems etc.), the old keys in the cache stay valid.

```

Reply:
Done, thanks for the hint!